core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXUpgradeV1Controller.java

```diff
@@ -80,6 +87,13 @@ public DistroXUpgradeV1Response upgradeClusterByCrnInternal(@ResourceCrn String
                 dataHubOsUpgradeEntitled));
     }
 
+    @Override
+    @InternalOnly
+    public DistroXCcmUpgradeV1Response upgradeCcmByCrnInternal(@NotEmpty @ResourceCrn String crn,  @InitiatorUserCrn @NotEmpty String initiatorUserCrn) {
```

Comment:
Please annotate `crn` with `@ValidCrn(resource = CrnResourceDescriptor.DATAHUB)` as well. Validation annotations must appear in both the endpoint and the controller.